dev-php/symfony-string/symfony-string-5.4.19.ebuild

```diff
@@ -26,9 +25,10 @@ RDEPEND="
 S="${WORKDIR}/string-${PV}"
 
 src_prepare() {
-	default
+	defaults
```

Comment:
I guess this slept through by accident, didn't it?

Reply:
Not sure how this happened. Should be fixed.